subprojects/test-runner/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/TestRunEnvironment.kt

```diff
@@ -150,18 +144,6 @@ internal fun parseReportDestination(argumentsProvider: ArgsProvider): ReportDest
     }
 }
 
-private fun outputDir(): File {
-    return if (Build.VERSION.SDK_INT >= 30) {
-        // public dir which is accessible by adb
-        InstrumentationRegistry.getInstrumentation().targetContext.externalMediaDirs[0]
-    } else {
-        ContextCompat.getExternalFilesDirs(
-            InstrumentationRegistry.getInstrumentation().targetContext,
-            null
-        )[0]
-    }
-}
```

Comment:
Moved to `RuntimeApplicationDirProvider`. 
See no need to know it here, because it depends only on Context but not from instrumentation arguments.